data/json/construction.json

```diff
@@ -1593,6 +1593,22 @@
     "pre_special": "check_empty",
     "post_terrain": "t_palisade_pulley"
   },
+  {
+    "type": "construction",
+    "id": "constr_palisade_pulley_wood",
+    "group": "build_rope_pulley_system",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 5 ], [ "mechanics", 2 ] ],
+    "time": "60 m",
+    "components": [
+      [ [ "rope_makeshift_30", 1 ], [ "rope_30", 1 ], [ "vine_30", 1 ] ],
```

Comment:
This can probably use a cordage requirement instead, see requirments there https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/requirements/materials.json

Reply:
Done.